awscli/testutils.py

```diff
@@ -783,12 +788,14 @@ def put_object(self, bucket_name, key_name, contents='', extra_args=None):
         if extra_args is not None:
             call_args.update(extra_args)
         response = client.put_object(**call_args)
+        # variable response assigned to but never used
```

Comment:
Again, I would prefer this gets removed rather than adding a comment. As is this doesn't actually fix the issue.